### ISSUE TITLE
Plugin is unfunctional when loaded on Minecraft 1.18.2.

### ISSUE THREAD

**Issue by RealTriassic** [DecentSoftware-eu/DecentHolograms#33]
### Description
When attempting to use the plugin on Minecraft 1.18.2, 2 errors are thrown to console that prevent you from doing so.

### Error Logs
```
[07:36:47 INFO]: [DecentHolograms] Enabling DecentHolograms v2.2.9
[07:36:47 ERROR]: Error occurred while enabling DecentHolograms v2.2.9 (Is it up to date?)
java.lang.ExceptionInInitializerError: null
        at eu.decentsoftware.holograms.api.DecentHolograms.enable(DecentHolograms.java:57) ~[DecentHolograms-2.2.9.jar:?]
        at eu.decentsoftware.holograms.api.DecentHologramsAPI.onEnable(DecentHologramsAPI.java:20) ~[DecentHolograms-2.2.9.jar:?]
        at eu.decentsoftware.holograms.plugin.DecentHologramsPlugin.onEnable(DecentHologramsPlugin.java:23) ~[DecentHolograms-2.2.9.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:501) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugin(CraftServer.java:559) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugins(CraftServer.java:473) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:671) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:438) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:352) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1172) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:322) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "eu.decentsoftware.holograms.api.utils.reflect.Version.getMinor()" because "eu.decentsoftware.holograms.api.utils.reflect.Version.CURRENT" is null
        at eu.decentsoftware.holograms.api.utils.reflect.Version.afterOrEqual(Version.java:62) ~[DecentHolograms-2.2.9.jar:?]
        at eu.decentsoftware.holograms.api.nms.NMS.<clinit>(NMS.java:40) ~[DecentHolograms-2.2.9.jar:?]
        ... 14 more
[07:36:47 INFO]: [DecentHolograms] Disabling DecentHolograms v2.2.9
[07:36:47 ERROR]: Error occurred while disabling DecentHolograms v2.2.9 (Is it up to date?)
java.lang.NullPointerException: Cannot invoke "eu.decentsoftware.holograms.api.nms.PacketListener.destroy()" because "this.packetListener" is null
        at eu.decentsoftware.holograms.api.DecentHolograms.disable(DecentHolograms.java:92) ~[DecentHolograms-2.2.9.jar:?]
        at eu.decentsoftware.holograms.api.DecentHologramsAPI.onDisable(DecentHologramsAPI.java:28) ~[DecentHolograms-2.2.9.jar:?]
        at eu.decentsoftware.holograms.plugin.DecentHologramsPlugin.onDisable(DecentHologramsPlugin.java:37) ~[DecentHolograms-2.2.9.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:266) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:401) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:538) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:376) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:501) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugin(CraftServer.java:559) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugins(CraftServer.java:473) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:671) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:438) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:352) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1172) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:322) ~[purpur-1.18.2.jar:git-Purpur-1574]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

### Information
Server Software: Purpur 1.18.2 (Build 1574)
Java Version: Java 17.0.2

**Comment by Andre601** [comment]
The issue is most likely that some NMS has changed in 1.18.2 and DH simply wasn't updated yet to (literally) reflect that.

**Comment by d0by1** [comment]
Fixed in the latest update 2.3.0.
